Sources/MapboxMaps/Ornaments/OrnamentsManager.swift

```diff
@@ -7,9 +7,9 @@ public enum OrnamentPosition: String, Equatable {
     case topLeft
     @available(*, deprecated, renamed: "topTrailing")
     case topRight
-    @available(*, deprecated, renamed: "bottomLeading")
-    case bottomRight
     @available(*, deprecated, renamed: "bottomTrailing")
+    case bottomRight
```

Comment:
let's double check if this change is indeed a mistake when rebasing/merge to main.